bench/bench_eb.c

```diff
@@ -153,13 +153,13 @@ static void util(void) {
 }
 
 static void arith(void) {
-	eb_t p, q, r, t[EB_TABLE_MAX];
+	eb_t p, q, r, t[EB_RELIC_TABLE_MAX];
```

Comment:
Why was RELIC_ added in between instead at the beginning here, i.e. RELIC_EB_TABLE_MAX?